winui/BusyIndicator/Overview.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title: About WinUI BusyIndicator control| Syncfusion
+description: Learn here all about introduction of Syncfusion WinUI BusyIndicator Control (SfBusyIndicator) with key features and more.
```

Comment:
description: Learn all about the introduction of Syncfusion WinUI BusyIndicator Control (SfBusyIndicator) with key features and more here.

Reply:
Changed the content as mentioned above.